app/src/main/java/com/example/indoor_volleyball/Models/Event.java

```diff
@@ -23,17 +26,15 @@ public class Event extends ParseObject {
     public static final String KEY_ENDTIME = "endTime";
     public static final String KEY_SKILLLEVEL = "skillLevel";
     public static final String KEY_DETAILS = "details";
-    public static final String KEY_QRCODE = "qrCode";
```

Comment:
QR code is a good idea for later if you do the code thing.  Easier than typing it in and an good opportunity to use an external library